src/components/modals/CategoryModal.tsx

```diff
@@ -140,10 +141,10 @@ export function CategoryModal(props: Props) {
               value={subcategory}
             />
           </View>
-          <FlatList
-            style={styles.resultList}
+          <FlashList
+            contentContainerStyle={styles.resultList}
```

Comment:
The top border is part of this CSS. Putting it on the scrolling content container will mean that the border scrolls away, which we probably don't want.